Makefile

```diff
@@ -1,20 +1,43 @@
 ARTIFACT_ID=k8s-longhorn
 VERSION=1.4.1-1
-MAKEFILES_VERSION=7.5.0
+MAKEFILES_VERSION=7.10.0
 
 include build/make/variables.mk
-include build/make/k8s.mk
 include build/make/clean.mk
 include build/make/self-update.mk
 
 ##@ Release
 
+K8S_PRE_GENERATE_TARGETS=generate-release-resource
+include build/make/k8s.mk
+
 .PHONY: generate-release-resource
 generate-release-resource: $(K8S_RESOURCE_TEMP_FOLDER)
 	@cp manifests/longhorn.yaml ${K8S_RESOURCE_TEMP_YAML}
 
-
 .PHONY: longhorn-release
 longhorn-release: ## Interactively starts the release workflow.
 	@echo "Starting git flow release..."
-	@build/make/release.sh longhorn
\ No newline at end of file
+	@build/make/release.sh longhorn
+
+
+# These targets wrap the helm targets because helmify always adds the project name as prefix to all resources.
+# This behaviour leads to wrong resource names because the original resource names do not start with the project name prefix.
+HELM_TEMPLATE_DIR=$(K8S_RESOURCE_TEMP_FOLDER)/helm/templates
+
+.PHONY: longhorn-k8s-helm-generate
+longhorn-k8s-helm-generate: k8s-helm-generate delete-longhorn-namespace
+
+.PHONY: delete-longhorn-namespace
+delete-longhorn-namespace:
+	@sed -i '0,/^# Source: deploy\/podsecuritypolicy.yaml$//d' ${HELM_TEMPLATE_DIR}/${ARTIFACT_ID}_${VERSION}.yaml
+
+.PHONY: longhorn-k8s-helm-apply
+longhorn-k8s-helm-apply: longhorn-k8s-helm-generate ## Generates and installs the helm chart.
+	@echo "Apply generated helm chart"
+	@${BINARY_HELM} upgrade -i ${ARTIFACT_ID} ${K8S_HELM_TARGET}
```

Comment:
Maybe add "--namespace <NAMESPACE_FROM_ENV>" and "--create-namespace" for local-deployment